### PR TITLE
Fixes conflicts with the woocommerce plugin

### PR DESCRIPTION
**Pull request by captn3m0**
- Both were using razorpay_payment_id as the form post hook
  which causes conflicts as QPP code runs when it should not
- This fixes it by adding a new post param on the form and using
  that for checks instead

### CODE REVIEW

File: frontend/checkout.phtml [hunk 1]
```diff
@@ -78,6 +78,7 @@
 <form name='razorpayform' id="paymentform" action="#redirectUrl#" method="POST">
     <input type="hidden" name="razorpay_payment_id" id="razorpay_payment_id">
     <input type="hidden" name="razorpay_signature"  id="razorpay_signature" >
+    <input type="hidden" name="rzp_QP_form_submit" value="1">
```
Comment (mayankamencherla) [inline frontend/checkout.phtml]: @captn3m0 why is the value set to 1?

File: razorpay-quick-payments.php [hunk 2]
```diff
@@ -275,15 +275,17 @@ function wpCheckRazorpayResponse()
 
         protected function getPostAttributes()
         {
-            $attributes = array(
-                'razorpay_payment_id' => sanitize_text_field($_POST['razorpay_payment_id']),
-                'razorpay_order_id'   => $_SESSION['razorpay_order_id'],
-                'razorpay_signature'  => sanitize_text_field($_POST['razorpay_signature'])
-            );
+            if (isset($_POST['rzp_QP_form_submit']))
```
Comment (mayankamencherla) [inline razorpay-quick-payments.php]: @captn3m0 Strict check?

Comment (captn3m0) [reply razorpay-quick-payments.php]: isset is gauranteed to return a boolean so I think it is fine.

**Comment by mayankamencherla** [comment]
@captn3m0 Approved

**Pull request merged**